openedx/core/djangoapps/user_authn/api/views.py

```diff
@@ -60,17 +60,17 @@ def get(self, request, **kwargs):  # lint-amnesty, pylint: disable=unused-argume
 
         if settings.ENABLE_DYNAMIC_REGISTRATION_FIELDS:
             if is_register_page:
-                registration_fields = RegistrationFieldsContext()._get_fields()  # pylint: disable=protected-access
+                registration_fields = RegistrationFieldsContext().get_fields()  # pylint: disable=protected-access
```

Comment:
You can remove the pylint disable command now that we have made it a non protected member of the class
```suggestion
                registration_fields = RegistrationFieldsContext().get_fields()
```